api/model-server-deprecated.yaml

```diff
@@ -0,0 +1,226 @@
+# WARNING - EXPERIMENTAL
+# This file was auto generated from the existing API using an IntelliJ plugin,
+# see https://www.jetbrains.com/help/idea/openapi.html#generate_openapi
+#
+# Manual changes were done for this 'spec' to work in-place with the
+# model-server for now. A lot of changes were done and are still necessary
+# to make this OpenAPI a viable artifact. It will most likely be split
+# into multiple OpenAPI files.
+
+openapi: "3.0.3"
+info:
+  title: "model-server deprecated API"
+  description: "modelix deprecated API"
+  version: "0.0.1"
```

Comment:
unfortunately not. ideally we should use the automatic semantic versioning here. but I would realize this in the build step during API publication.

Reply:
No, not at all. Just because we do a major increment in the modelix.core version doesn't mean we have a breaking change in the REST API. It would be a real problem with this project structure to go that way. We would constantly indicate to clients that potentially have changed in a breaking fashion.

Reply:
makes sense. I put it to `1.0.0` for now